docs/getting_started_on_linux.md

```diff
@@ -68,6 +68,42 @@ environment variable is set when you are building. You may find it useful to add
 `source [PATH TO VULKAN SDK]/setup-env.sh` to your `~/.bashrc` file to simplify
 environment variable setup.
 
+## Optional: Configure Git
+
+### Git SSH
+
+*   Generate SSH Key: `ssh-keygen -t rsa -b 4096 -C "EMAIL@email.com"`
+*   Add `~/.ssh.id_rsa.pub` key to GitHub
+*   Try a test connection `ssh git@github.com`
+
+### Other git config options
+
+```shell
+git config --global user.name "MY NAME"
+git config --global user.email "MY EMAIL"
+```
+
+### Clone
+
+This assumes that we are cloning into `/home/user/ireepub`. Update accordingly for
```

Comment:
ireepub seems like an unlikely directory name for an external contributor to use and "user" is unlikely to be their username. What about using an env variable that references "$HOME".

Also I think I would say that we're cloning *into* `/home/user/ireepub/iree` in this case. What about "cloning under"

```suggestion
This assumes that we are cloning under `$HOME/git`. Update accordingly for
```

Looks like there's no way to do multi-line suggested changes (what's even the point then?), but that would make the below shell block


```shell
BASE_DIR="${HOME?}/git"
mkdir -p ${BASE_DIR?}
cd ${BASE_DIR?}
git clone $IREE_CLONE_ARGS https://github.com/google/iree.git iree
cd iree
git submodule init
git submodule update $IREE_CLONE_ARGS --recursive
```

Reply:
Why is base_dir "${HOME?}/git"

Reply:
`ireepub` is what is used on windows side.

Reply:
Oh huh, weird... Ok let's keep it consistent then. Not sure I understand the question, but it's just an arbitrary subdirectory under home (~). It happens to be where I stick my git repositories. I don't actually care what we put here (what do most people do?), but I think we should use `$HOME` instead of `/home/user` just to make copy-pasting this a bit easier.